introduction.mdx

```diff
@@ -9,10 +9,10 @@ mode: "wide"
 ## What is Hypermode? {/* <!-- vale Google.Contractions = NO --> */}
 
 Hypermode is a managed service that provides the workbench and infrastructure to
-create **powerful, engaging, and secure AI assistants, APIs, and backend
-services**.
+create **powerful, engaging, and [secure](/security) AI assistants, APIs, and
```

Comment:
I wouldn't link to the service's Security page here, as that doesn't cover the security architecture of the Modus app that is more directly being addressed here

Reply:
Okay! I'll remove the link